packs/menace-under-otari-bestiary/warg-bb.json

```diff
@@ -0,0 +1,518 @@
+                "description": {
+                    "value": "<p>When the warg succeeds at its jaws Strike, it can use this action immediately after to attempt to @UUID[Compendium.pf2e.actionspf2e.Item.Grapple] the target. This Grapple doesn't apply or count toward the warg's multiple attack penalty.</p>"
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [],
+                "slug": null,
+                "traits": {
+                    "value": []
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "KapyaNm62BvUi1J6",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Pack Attack",
+            "sort": 500000,
+            "system": {
+                "actionType": {
+                    "value": "passive"
+                },
+                "actions": {
+                    "value": null
+                },
+                "category": null,
+                "description": {
+                    "value": "<p>The warg's jaws Strike deals [[/r 1d4]] extra damage to creatures within reach of two of the warg's allies.</p>"
+                },
+                "publication": {
+                    "license": "OGL",
+                    "remaster": false,
+                    "title": ""
+                },
+                "rules": [
+                    {
+                        "domain": "damage",
+                        "key": "RollOption",
+                        "option": "pack-attack",
+                        "toggleable": true
+                    },
+                    {
+                        "diceNumber": 1,
+                        "dieSize": "d4",
+                        "key": "DamageDice",
+                        "predicate": [
+                            "pack-attack"
+                        ],
+                        "selector": "strike-damage"
+                    }
+                ],
+                "slug": null,
+                "traits": {
+                    "value": []
+                }
+            },
+            "type": "action"
+        },
+        {
+            "_id": "Db24UdpGiWsgWITM",
+            "img": "systems/pf2e/icons/default-icons/action.svg",
+            "name": "Shred",
+            "sort": 600000,
+            "system": {
+                "actionType": {
+                    "value": "action"
+                },
+                "actions": {
+                    "value": 1
+                },
+                "category": null,
+                "description": {
+                    "value": "<p>If the warg has a creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed], it can spend one action to shred them with its teeth. The creature takes [[/r 1d6[slashing]]] damage and can make a @Check[type:fortitude|dc:18|basic:true] save.</p>"
```

Comment:
```suggestion
                    "value": "<p>If the warg has a creature @UUID[Compendium.pf2e.conditionitems.Item.Grabbed], it can spend one action to shred them with its teeth. The creature takes @Damage[1d6[slashing]] damage and can make a @Check[type:fortitude|dc:18|basic:true] save.</p>"
```